model/src/main/kotlin/utils/CopyrightStatementsProcessor.kt

```diff
@@ -19,11 +19,13 @@
 
 package com.here.ort.model.utils
 
+import com.here.ort.model.CopyrightFindings
```

Comment:
Commit message:

> Also remove the `LicenseFindingsMap.processStatements()` method because it is not compatible anymore

Compatible to what? How about saying "because it can only operate on Strings instead of CopyrightFindings"?

> and temporarily disable the copyrights processing in the notice reporter.

Could you briefly explain *why* it is necessary to temporarily disable it (e.g. instead of fixing / refactoring it in one go)?

Reply:
Done.